wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -21,52 +21,33 @@ import CoreLocation
 import Contacts
 import WireDataModel
 import WireCommonComponents
+import SwiftUI
 
 class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
 
     // MARK: - Properties
     typealias LabelColors = SemanticColors.Label
+    private typealias deviceDetailsSection = L10n.Localizable.Device.Details.Section
```

Comment:
```suggestion
    private typealias DeviceDetailsSection = L10n.Localizable.Device.Details.Section
```
We usually create typealias with capital letters.